src/main/webapp/app/entities/student-with-team.model.ts

```diff
@@ -0,0 +1,7 @@
+export class StudentWithTeam {
```

Comment:
please add some documentation what this class is used for.
Also explain that developers can either use username or registrationNumber as unique identifier and that one of those values must be set